<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M_SetDev</class>
 <widget class="QWidget" name="M_SetDev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>硬件参数</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>2</y>
     <width>261</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="title">
    <string>矢量网络分析仪</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_power">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>中频带宽：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>dBm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>发射功率：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_width">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_freq">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>43</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>59</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>端口功率:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_port_power">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>135</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>dBm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>dBm</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButto_OK">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>210</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>140</y>
     <width>261</width>
     <height>57</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="title">
    <string>衰减器</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>31</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>衰减量：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_attan">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>29</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>dBm</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_default">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>默认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pulse">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>脉冲</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButto_apply">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>210</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sweep_time">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>349</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>345</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>扫描时间：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Agilent</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>R&amp;S</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>仪器类型：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
